### PR TITLE
Format variadic arguments

### PR DESCRIPTION
**Pull request by marcusklaas**
Closes https://github.com/nrc/rustfmt/issues/441.

### CODE REVIEW

File: src/items.rs [hunk 1]
```diff
@@ -506,27 +508,62 @@ impl<'a> FmtVisitor<'a> {
...
+                Some(ArgumentKind::Variadic)
+            } else {
+                None
+            };
+            let variadic_start = if variadic {
+                let span = codemap::mk_sp(args.last().unwrap().ty.span.hi, span.hi);
+                span_after(span, "...", self.codemap) - BytePos(1)
+            } else {
+                BytePos(0)
+            };
```
Comment (nrc) [inline src/items.rs]: Could you add the start byte pos to the Variadic enum variant? Then you only need one if/else here and you don't need the magic 0 for variadic_start

File: tests/target/extern.rs [hunk 2]
```diff
@@ -27,3 +27,10 @@ extern "Rust" {
     // Some comment.
     pub static mut var: SomeType;
 }
+
+extern {
+    fn syscall(number: libc::c_long, // comment 1
+               // comm 2
+               ... /* sup? */)
+               -> libc::c_long;
+}
```
Comment (nrc) [inline tests/target/extern.rs]: Could you add a test without comments too please?

**Comment by nrc** [comment]
r+ with those changes

File: src/items.rs (hunk 1 above)
Comment (marcusklaas) [inline src/items.rs]: That's much better. Thanks for the tip!

**Pull request merged**